gensim/utils.py

```diff
@@ -266,6 +266,10 @@ def load(cls, fname, mmap=None):
         obj = unpickle(fname)
         obj._load_specials(fname, mmap, compress, subname)
         logger.info("loaded %s", fname)
+        if cls.__name__ is 'Similarity':
```

Comment:
`issubclass` better?

Also, comparing strings for object equality with `is`, instead of content equality with `==`, is undesirable. Python string interning is a non-trivial subject.